OpenKh.Tools.ModsManager/Services/ConfigurationService.cs

```diff
@@ -93,11 +97,49 @@ static ConfigurationService()
                     .ToList();
             });
         }
-
+        public static string LaunchGame
+        {
+            get => _config.LaunchGame;

```

Comment:
I have made ConfigurationService default to "kh2" to avoid this error. Tested with a new setup and making sure it also works on Mod Managers that have already gone through first time setup.